### DISCUSSION TITLE
Lock the app - Privacy PIN/Face ID

### DISCUSSION THREAD

**Discussion by JonoHall** [aeharding/voyager#158] [Ideas]
I really appreciated the lock on Apollo's app locking feature.

When you exited or switched away from Apollo it would immediately lock/blur the app.

Is this something that can still be added when it's not a full app?

**Comment by aeharding** [comment]
I'm not sure if it will blur in the app switcher (would need to investigate) - but a lock is doable.

It couldn't be trusted as actual security, though.

**Reply by JonoHall** [reply]
May not be trusted, but anything would be a great stopgap to stop unintended viewing of the app and its.. _content_.

May thanks for looking into this!

/ps. Absolutely loving the app, and exciting to see the number of updates being pushed out! Please don't burn yourself out though hah!

**Comment by JonoHall** [comment]
> I'm not sure if it will blur in the app switcher

I wonder if Apollo simply added a blur overlay immediately before asking iOS to lock the app, instead of relying on an Apple switcher blur feature (which may or may not exist).
